test/server/browser-provider-test.js

```diff
@@ -426,6 +427,24 @@ describe('Browser provider', function () {
         });
     });
 
+    describe('Features', () => {
+        const PROVIDERS_WITH_MULTIWINDOW_MODE = ['chrome', 'chromium', 'chrome-canary', 'edge', 'firefox'];
```

Comment:
I just wonder should 'edge' be in this array? It contradicts with the title, first comment of the issue and the branch name.

Reply:
The dedicated `edge` provider handles the Chromium-based that works the same way as Chrome. So the test is correct. The title should say Chromium-based instead of Chrome.

Reply:
And there should be Safari instead of Edge in the branch name.